dashboard/src/shared/KubelessConfig.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+import { IKubelessConfigMap } from "./types";
+
+export default class Config {
+  public static async get() {
+    const config = await axios.get<IKubelessConfigMap>(Config.SelfLink);
```

Comment:
I think this should return config.data, or better yet could be written as `const { data: config } = ...`. There's not much point in returning the Axios request response.